server/config.ts

```diff
@@ -76,8 +85,8 @@ export default {
         deadline: Number(get('HMPPS_AUTH_TIMEOUT_DEADLINE', 10000)),
       },
       agent: new AgentConfig(Number(get('HMPPS_AUTH_TIMEOUT_RESPONSE', 10000))),
-      authClientId: get('AUTH_CODE_CLIENT_ID', 'clientid', requiredInProduction),
-      authClientSecret: get('AUTH_CODE_CLIENT_SECRET', 'clientsecret', requiredInProduction),
+      authClientId: get('API_CLIENT_ID', 'clientid', requiredInProduction),
```

Comment:
The typescript template was updated to use `authClientId` instead of `apiClientId` so I think we should keep with that naming for consistency: https://github.com/ministryofjustice/hmpps-template-typescript/blob/223c4bca8509bd04806d1b57026f9970a8063c67/CHANGELOG.md?plain=1#L39